test-end-to-end/android-maestro/common/course_with_members.yaml

```diff
@@ -0,0 +1,87 @@
+appId: com.toughra.ustadmobile
```

Comment:
Let's call this file create_course_with_members - because creating the course is what it does.

Reply:
Sure Mike